src/pages/detail/AmenitiesComponent.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import {
+  WifiIcon,
+  AdjustmentsIcon,
+  DesktopComputerIcon,
+  CogIcon,
+} from '@heroicons/react/outline'
+
+const AmenitiesComponent = ({ lodgingData }) => {
+  const [expanded, setExpanded] = useState(false)
+
+  const getIconComponent = (amenityName) => {
+    switch (amenityName) {
+      case '와이파이':
+        return <WifiIcon className="h-5 w-5 text-gray-500" />
+      case '에어컨':
+        return <AdjustmentsIcon className="h-5 w-5 text-gray-500" />
+      case 'TV':
+        return <DesktopComputerIcon className="h-5 w-5 text-gray-500" />
+      // Add cases for other amenities
+      default:
+        return <CogIcon className="h-5 w-5 text-gray-500" /> // Placeholder for default case
+    }
+  }
+
+  return (
+    <div className="bg-white p-6 rounded-lg shadow-md mt-4 mb-4">
+      <h3 className="text-xl font-semibold mb-4">숙소 편의시설</h3>
+      <ul className={`${expanded ? '' : 'max-h-24 overflow-hidden'}`}>
```

Comment:
아래와 같이하는게 더 보기 좋을 것 같습니다.
```suggestion
      <ul className={`${expanded && 'max-h-24 overflow-hidden'}`}>
```